### PR TITLE
release/0.9

### PR DESCRIPTION
**Pull request by sils**
Upon merge of this PR I will perform the actual release from the release commit.

### CODE REVIEW

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
+           '    :MM             .- ,
+            '.   `#?..  .       ..'
+               -.   .         .-
+                 ''-.oo,oo.-''
+
+For this release only we had `58 different contributors
+<http://pastebin.com/raw/PpdZm7yL>`_ from all around the globe contributing way
+over 200 commits over 2.5 months to coala.
+
+**The name of this release is GlobalBear** to honour our `GlobalBear` class and
```
Comment (fneu) [inline RELEASE_NOTES.rst]: "honor" would be American English

File: RELEASE_NOTES.rst [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+leave a statement on how global the community grows: gone are the days when we
+visit conferences and we have to explain the project to all of the people. More
```
Comment (fneu) [inline RELEASE_NOTES.rst]: "all the people" without "of

File: RELEASE_NOTES.rst [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+and more people know the project before we meet them and this is great! This is
+a huge step in our conquest to take over the world!
+
+The `GlobalBear` class serves our users by providing project wide ("global")
```
Comment (fneu) [inline RELEASE_NOTES.rst]: parentheses or quotes?

File: RELEASE_NOTES.rst [hunk 4]
```diff
@@ -1,5 +1,57 @@
...
+analysis. This release it will probably make its last appearence because it
```
Comment (fneu) [inline RELEASE_NOTES.rst]: appearance

File: RELEASE_NOTES.rst [hunk 5]
```diff
@@ -1,5 +1,57 @@
...
+will be deprecated in favour of a `more sophisticated concept
```
Comment (fneu) [inline RELEASE_NOTES.rst]: favor would be the American term again

File: RELEASE_NOTES.rst [hunk 6]
```diff
@@ -1,5 +1,57 @@
...
+<https://github.com/coala/cEPs/blob/master/cEP-0002.md>`_ in the near future.
+
+We have also worked a lot towards building our dream of **letting users declare
+code analysis configuration completely language independently**: to take a
+sneak peak at what we want to do, `check this out
```
Comment (fneu) [inline RELEASE_NOTES.rst]: "sneak peek"! It's not a crawling mountain :D

File: RELEASE_NOTES.rst [hunk 7]
```diff
@@ -1,5 +1,57 @@
...
+the user a plethora of facts around the type of issue pointed out.
+
+**For users**, we have added a lot of usability improvements as well as for example
+the ability to merge patches within one line: if you previously had to run
+coala multiple times because of patch conflicts, this is likely not the case
+anymore!
+
+**As a Bear writer** you now have access to our ``Language`` facilities: they
+will give you facts about programming languages that you analyse so you can
+build truly language independent bears. Also you now can use
```
Comment (fneu) [inline RELEASE_NOTES.rst]: comma after "Also" while we're at it ;)

**Comment by fneu** [comment]
ack https://github.com/coala/coala/pull/3098/commits/9af80017e10014061d21b2674dbcf6e1766c7fa6, https://github.com/coala/coala/pull/3098/commits/9420ad5473d1c601f93b164f56016fae7de8500d

Although this is kind of a weird scheme tbh. Are you sure that everybody (pip, pypi, ...) understands that 0.9.0 > 0.9.0.devXXXX ? Might be me being off here, though

**Comment by sils** [comment]
yes, I'm sure pypi understands that. We're doing this since quite some time and it's their spec btw.

**Comment by sils** [comment]
9004f61 VERSION: Bump to development
7b87fdb Release coala 0.9 GlobalBear
reack

**Comment by fneu** [comment]
I know we're doing it, I just didn't know why ;)

**Comment by sils** [comment]
fixed the typos :)

File: RELEASE_NOTES.rst [hunk 8]
```diff
@@ -1,5 +1,57 @@
-coala 0.9.0
-===========
+coala 0.9.0 - GlobalBear
+========================
+
+::
```
Comment (fneu) [inline RELEASE_NOTES.rst]: not sure about rst syntax in general, but this is messed up when viewed on GitHub.
Needs a new line after `::`?

**Comment by sils** [comment]
reack
1928a8d VERSION: Bump to development
e6ad24d Release coala 0.9 GlobalBear

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala.io/ - http://viperdev.io/ - http://gitmate.com/

**Comment by fneu** [comment]
ack https://github.com/coala/coala/pull/3098/commits/d2f71d8568ae0414044630f9f58eed8bd6d0a558

**Comment by Nosferatul** [comment]
ack 1928a8d

**Comment by aptrishu** [comment]
ack 1928a8d

**Comment by sils** [comment]
@rultor merge

**Pull request merged**